meetings/breakout_meetings/2019-01-14-project-progression.md

```diff
@@ -0,0 +1,101 @@
+* Myles: We’re talking about becoming a top level project.
+* Michael: Could you join as a BDFL and join, and stay that way?
+* Myles: From the legal standpoint there will be clear guidelines for participation as you graduate to different levels. We have a vision of what a sustainable open governance model looks like, and if you want to join through our process, we can guide you through that process.
+* Dave: When you get to legal issues it gets a bit trickier, but if you look at the JSF lists for example - we do have a membership process. The period of time that you’re not a voting member could be very short if you want to be active and be a member of CPC. You can follow a best practice to get there.
+* Jory: My Concerns for these definitions is that you might be precluding these smaller projects from having a voice and being able to take on voting decisions.
+* Matteo: For me it seems a little bit complicated. If a project isn’t very active, would they be willing to invest time in building the foundation?
+* Jory: There’s a difference between thriving and small projects. There are upper limits to projects, though they are thriving, and that’s ok.
+* Michael: We’re trying to word it so anyone can participate in the regular work. There are multiple open paths to becoming voting members, if you’re big enough - get it, but if you’re smaller, you can get recognition as voting members by the many paths we have to get there.
+* Jory: We believe in the power of small projects/business and what they can bring to the table. We don’t want to undervalue that.
+* Tierney: I concur 100%. That’s how node was built.
+* Myles: not hearing anything that’s saying small projects can’t become voting members/contributors right now as a top level project.
+* Jory: If a project needs to have an open governance and we don’t really know what an open governance is, we’re stuck. We need a definitions of what that means here in this meeting.
+* Myles: We can extrapolate features from our existing projects on what that means.
+* Jory: I’ve talk with projects about where they want to be with us, I don’t think there will be too many hang ups. It just comes down to being top level voting members.
+* Jory: Projects who’ve expressed interest in becoming top level: Web Hint, Dojo, JerryScript, WebDriver, Mocha.
+* Dave: Having true open governance requires enough people regularly participating, and enough diversity in the contribution base, etc. We need mentorship to exist.
+* Matteo: I can stand behind the JSF’s mentorship values 100%.
+
+### 15 mins - Defined Project Proposal Requirements, & explicit metrics for how to validate that those requirements have been met. (Chris H., Matteo, Jory)
+
+- do we require projects to keep public lists of orgs using their technology?
+
+
+### 10 mins - Clarification of Stage Definitions and Monikers (see also #44)
+
```

Comment:
Maybe add a note that we didn't get to this?

Reply:
okiedok